src/test/kotlin/org/malachite/estella/organization/OrganizationIntegration.kt

```diff
@@ -87,6 +88,37 @@ class OrganizationIntegration : BaseIntegration() {
 
     @Test
     @Order(6)
+    fun `should return ok with list of offers`() {
+        val response = getOrganizationsOffers(legitOrganizationUser.mail, legitOrganizationPassword)
+        response.forEach{
+            it.let {
+                expectThat(it.organization.name).isEqualTo(legitOrganizationName)
+            }
+        }
+    }
+
+    private fun getOrganizationsOffers(mail: String, password: String): List<OfferResponse> {
```

Comment:
Jeśli to jest funkcja jednorazowego użytku, to musi być to funkcja?

Reply:
a nie może? ogólnie jak jest coś więcej w requeście do zrobienia niż dostanie responsa, to myślę że warto trochę granulować kod żeby było mniej szumu we właściwym teście

Reply:
Mi tu w sumie bardziej chodziło o to, że potem mamy w ciula funkcji i ciężko znaleźć tą, którą potrzebujesz (bo na przykład coś się tam popsuło i skaczesz od testu do funkcji).

Reply:
mamy IDE więc nie ma problemu z poruszaniem się po kodzie, jak to sam mi niedawno powiedziałeś, a  wydaje mi się że przejrzystość też jest ważna xd